### ISSUE TITLE
Problem with integration of inkscape-figures

### ISSUE THREAD

**Issue by greyback2305** [sleepymalc/VSCode-LaTeX-Inkscape#18]
Hello, 
I have been trying to make this setup for the past few days, and now that it works pretty well for the first part, i wanted to integrate Inkscape. The only issue, it seems, is related to the integration of "inkscape-figures". I have downloaded successfully choose-gui and inkscape-figures, in the same way as intented, except that the command weren't working, so i changed a bit them, but it seemed to work. 
But just after installing it, i tried to type the command `inkscape-figures`, and it didn't work. I got `zsh: command not found: inkscape-figures`. 
Do i need to install the files in PATH ? I had an warning message when installing inkscape-figures  whith `pip3 install inkscape-figures` saying that nosetest and nosetest 3.7 weren't in path. 
I am totally new to coding so excuse my incompetence.
Thanks for the response

**Comment by sleepymalc** [comment]
Right, your computer finds executables under all the directories in your `$PATH` variable. So, in this case, seems like the `inkscape-figures` (an executable) is in a directory that is not in your `$PATH` variable. Just add it into your `$PATH` variable then you should be good to go.

**Comment by greyback2305** [comment]
Ok, so i have tried to add these files in path, by doing like this in my .bash_profile
<img width="666" alt="Capture d’écran 2023-01-12 à 22 00 17" src="https://user-images.githubusercontent.com/122486486/212180297-db1b7fe6-68d3-4009-ac7d-f9c62b49707d.png">

it didn't seemed to work, even after rebooting the terminal, so i also tried to put directly the `inkscapefigures` folder directly in path. 
<img width="735" alt="Capture d’écran 2023-01-12 à 21 51 05" src="https://user-images.githubusercontent.com/122486486/212180545-7352864c-0687-46ea-b23f-3823fe4245e8.png">
This was the message i got when installing `inkscape-figures`, so i added these files in path, like said before. 
When i type the command "inkscape-figures" in the terminal i got this (i put it here even if it does not seem useful.
<img width="342" alt="Capture d’écran 2023-01-12 à 21 50 43" src="https://user-images.githubusercontent.com/122486486/212180879-c3f600ca-ff63-43a5-bdde-665904d94bd5.png">

**Comment by sleepymalc** [comment]
Hmm, maybe try `inkscape-figures` instead of `inkscapefigures`? The former is the executable's name, while the latter (where you seemed to be typing in the terminal) is not.

**Comment by greyback2305** [comment]
yeah, that was the wrong screenshot, i got the same error for typing `inscape-figures`

**Comment by sleepymalc** [comment]
I guess the first thing you should do is to find where is `inkscape-figures` installed, and try to run it via terminal to make sure it's correctly installed.

**Comment by greyback2305** [comment]
So `inkscape-figures` seems to be located in the intended spot
<img width="711" alt="Capture d’écran 2023-01-14 à 19 01 52" src="https://user-images.githubusercontent.com/122486486/212488363-7fc9b8f1-558a-432f-877a-916c179208ac.png">
I really don't understand why it isn't working but thanks for your help still

**Comment by sleepymalc** [comment]
Can you print out your `$PATH` variable? You can use `echo $PATH` to see what's in your `$PATH`.

**Comment by greyback2305** [comment]
<img width="828" alt="Capture d’écran 2023-01-14 à 22 45 31" src="https://user-images.githubusercontent.com/122486486/212498430-bf21f6bb-fdf9-4a79-b977-ba23f5ac8b2b.png">

**Comment by sleepymalc** [comment]
Yes, the problem is that `$PATH` will not do a recursion search, i.e., if your executable is under `/path/dir`, and you only have `/path` in your `$PATH`, then your system will not be able to find the executable except you add `/path/dir` explicitly.
So you should add exactly `Users/gabriel/Library/Python/3.8/lib/python/site-packages (1.0.8)` into your path.

**Comment by greyback2305** [comment]
<img width="614" alt="Capture d’écran 2023-01-15 à 11 01 38" src="https://user-images.githubusercontent.com/122486486/212534606-c6b1bf93-da97-447f-8156-b473ac98dc07.png">

So i added, like you said the accurate directory in `$PATH`, but again, `command not found` when i typed `inkscape-figures`. Here is `site-packages` in Finder, if you see something. Thanks again
<img width="421" alt="Capture d’écran 2023-01-15 à 11 02 10" src="https://user-images.githubusercontent.com/122486486/212534609-82ae8ccd-4608-4f3d-934a-efea556c6c98.png">

**Comment by sleepymalc** [comment]
You path should include your _**executable**_, i.e., `inkscape_figures`. For example, my `inkscape_figures` executable is in `Users/pbb/opt/anaconda3/bin`:
<img width="436" alt="image" src="https://user-images.githubusercontent.com/60427058/212569841-35b62008-2f00-403f-bdd4-a9fdd03afe30.png">
so my `$PATH` also includes this:
<img width="721" alt="image" src="https://user-images.githubusercontent.com/60427058/212569886-31e6fe2c-0d66-4bfc-b97a-b2f4894843e3.png">
Make sure you understand how that `$PATH` works.

**Comment by greyback2305** [comment]
ok, sorry for the wait, i had an exam last week. I just checked and i think that i understand now.  
I found the problem. It was only that the names of the files in my path had a ":" at the end. I don't understand why these were like this at first but i didn't paid attention when copy pasting the command in bash_profile lol. 
Thanks for your help still, that's very cool for the newbies reading your project.
